packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -635,22 +635,25 @@ class TestConfiguration {
     return this._req(view, null, controllers.view.v1.save)
   }
 
-  async createViewV2(config?: Partial<ViewV2>) {
-    if (!this.table) {
-      throw "Test requires table to be configured."
-    }
-    const view = {
-      tableId: this.table._id,
-      name: generator.guid(),
-      ...config,
-    }
-    return this._req(view, null, controllers.view.v2.save)
-  }
-
-  getViewV2(viewId: string): supertest.Test {
-    return this.request!.get(`/api/v2/views/${viewId}`)
-      .set(this.defaultHeaders())
-      .expect("Content-Type", /json/)
+  api = {
```

Comment:
Sorry for being unclear, in my original suggestion around using test api clients I meant in similar fashion as we do in worker and account portal e.g. 

https://github.com/Budibase/budibase/blob/9dfd4aa811479a1dda72eff7ba2588d66b132277/packages/worker/src/tests/api/index.ts#L21-L39

The views code would be a good first use case. Meaning we can extract api specific concepts like views and their operations out of the `TestConfiguration` file directly. 

I would also be in favour of not using `controllers.view.v2.save` directly, as any use of this will bypass middlewares and request / response body parsing which is a main concern of the api unit tests.

Reply:
Are you thinking to use the `superttest` to do api calls, or just calling the sdk directly?

Reply:
just the way we do it currently with `request.get`, `request.post` etc